dev/Breadcrumb/BreadcrumbBar.xaml

```diff
@@ -89,6 +101,7 @@
                                         <Setter Target="PART_ChevronTextBlock.Visibility" Value="Collapsed"/>
                                         <Setter Target="PART_SeparatorColumnDefinition.Width" Value="0px"/>
                                         <Setter Target="PART_ChevronColumnDefinition.Width" Value="0px" />
+                                        <Setter Target="PART_LayoutRoot.Margin" Value="{StaticResource BreadcrumbBarItemLastItemMargin}"/>
```

Comment:
>PART_LayoutRoot [](start = 56, length = 15)

It looks like you are setting the same property from two different VisualStateGroups. This can cause issues, because the system will take whatever the last state entered was instead of finding some appropriate value. Also the VSM system tracks where property changes come from, which can get messed up when  you have two state groups touching the same property.

Reply:
the Margin property can't be modified by the two states at the same time, by using the first one "EllipsisDropDown" we limit the code to only transition between the states located under "EllipsisDropDownItemCommonStates", while inline elements would only use "InlineItemTypeStates" states, where the "LastItem" state is located. So this should not generate any issue